tests/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -1869,46 +1869,94 @@ public void TestColumns()
 
         }
 
+        [Fact]
+        public void TestRows()
+        {
+            DataFrame df = MakeDataFrameWithAllColumnTypes(10);
+            DataFrameRowCollection rows = df.Rows;
+            Assert.Equal(10, rows.Count);
+            DataFrameRow firstRow = rows[0];
+            object firstValue = firstRow[0];
+            Assert.Equal(df[0, 0], firstValue);
+            long rowCount = 0;
+            foreach (DataFrameRow row in rows)
```

Comment:
How about testing enumerating through the objects in a `DataFrameRow`?

Reply:
Done.